cmd/platform/channel_test.go

```diff
@@ -44,6 +44,24 @@ func TestRemoveChannel(t *testing.T) {
 	checkCommand(t, "channel", "remove", th.BasicTeam.Name+":"+channel.Name, th.BasicUser2.Email)
 }
 
+func TestMoveChannel(t *testing.T) {
+	th := api.Setup().InitBasic()
+	defer th.TearDown()
+
+	channel := th.CreateChannel(th.BasicClient, th.BasicTeam)
+
+	adminEmail := th.BasicUser2.Email
+	origin := th.BasicTeam.Name + ":" + channel.Name
+	dest := th.BasicTeam2.Name
+
+	checkCommand(t, "channel", "add", origin, adminEmail)
+
+	// should fail with nill because errors are logged instead of returned when a channel does not exist
+	require.Nil(t, runCommand(t, "channel", "move", dest, th.BasicTeam.Name+":doesnotexist"))
+
+	checkCommand(t, "channel", "move", dest, origin)
```

Comment:
It might be wise to add an assert that checks that the header for channel that was moved was correctly modified as per line 1335 of `app/channel.go`

Reply:
that sounds like a good plan. I was debating if it was worth it to add some regex to prevent subsequent channel moves from continuously appending to the header. Also should I just commit to this branch or do another squash rebase and make a new PR?

Reply:
It's fine to just push more commits to this branch - we do a squash merge anyway when we merge PRs.

Reply:
@MusikPolice i am not longer modifying the header. will this test be sufficient in that case?

Reply:
I think so, yes.